index.js

```diff
@@ -95,6 +95,18 @@ module.exports = {
                   /components\/scroll-anchor/
                 ];
                 break;
+              case 'non-bg-vi':
```

Comment:
`bg-non-vi` would be better name (you are a BG app, but don't want of the `vi` part :)

Reply:
@Leooo - as we keep this under exclude, so can we just use `bg-vi`?

Reply:
@Leooo - Kept it `bg-non-vi` only :)

Reply:
@ssahabg maybe notations are not very clear, but here is the idea:

* the "bg" exclude bundle means: exclude everything which is already provided by bg internal libraries (bg-vi being one of those libraries)

so in your case you still want to exclude those bg libraries - except bg-vi

=> hence the `bg-non-vi` idea. `bg-except-bg-vi` would also work, for example. But just `bg-vi` wouldn't work because you have to exclude all the other `bg` libraries too. I hope it makes sense.

Reply:
@Leooo - I understand now. Hence, we can go with `bg-except-bg-vi`.

Reply:
ok